app/pb_kits/playbook/pb_form_pill/_form_pill.jsx

```diff
@@ -0,0 +1,74 @@
+/* @flow */
+import React from 'react'
+import classnames from 'classnames'
+import Title from '../pb_title/_title.jsx'
+import Icon from '../pb_icon/_icon.jsx'
+import Avatar from '../pb_avatar/_avatar.jsx'
+
+type FormPillProps = {
+
+className?: String,
+id?: String,
+text: String,
+name: String,
+onClick?: EventHandler,
+avatar?: Boolean,
+avatarUrl?: String,
+}
+const FormPill = ({
+  className,
+  text,
+  name,
+  avatar = false,
```

Comment:
This might cause unintended bugs - perhaps we can not add the prop if it is blank